modules/openapi-generator/src/test/resources/3_0/petstore-echo.yaml

```diff
@@ -0,0 +1,1972 @@
+openapi: 3.0.0
+info:
+  description: >-
+    This spec is mainly for testing Petstore server and contains fake endpoints,
+    models. Please do not use this for any other purpose. Special characters: "
+    \
+  version: 1.0.0
+  title: OpenAPI Petstore
+  license:
+    name: Apache-2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: http://localhost:3000/
+paths:
+  /foo:
+    get:
+      responses:
+        default:
+          description: response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  string:
+                    $ref: '#/components/schemas/Foo'
```

Comment:
not required as part of this PR.

instead of creating this petstore-echo spec, what about using echo apec instead: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/echo_api.yaml#L501 ?

can you migrate the test to use the Echo API later after this PR is merged?